qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -91,6 +91,11 @@ def basis_gates(self) -> List[str]:
         """Return the basis gates supported by the library."""
         return list(name for name in self._schedules)
 
+    @property
+    def init_default_values(self) -> Dict:
```

Comment:
I wonder if we generally should be trying to put some kind of serialize and deserialize (like `to_json` and `from_json`?) methods on our classes since we seem to want to move them in and out of the database.

Reply:
I think we should do that. Perhaps rather in a separate PR to keep changes relatively small?

Reply:
I changed this PR to implement this functionality. See here: ab58043 and here 2c7f617